locales/es.json

```diff
@@ -0,0 +1,39 @@
+{
+  "Back": "Volver",
+  "Newer Posts": "Las publicaciones más nuevas",
+  "Older Posts": "Publicaciones antiguas",
+  "Page {page} of {pages}": "Página {page} de {pages}",
+  "with the email address": "con la dirección de correo electrónico",
+  "Your email address": "Tu correo electrónico",
+  "You've successfully subscribed to": "Te has suscrito con éxito a",
+  "A collection of posts": "Una colección de publicaciones",
+  "A collection of 1 post": "Una colección de 1 publicación",
+  "A collection of % posts": "Una colección de % publicaciones",
+  "Get the latest posts delivered right to your inbox": "Recibe las últimas publicaciones entregadas directamente en tu bandeja de entrada",
+  "Latest Posts": "Últimas publicaciones",
+  "No posts": "No publicaciones",
```

Comment:
`No se han encontrado entradas` would be more like "No posts found"; is that the right context?

Reply:
That works better in context. it would be like a author page where the author hasn't written anything.